platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/service/impl/ApplicationServiceBean.java

```diff
@@ -402,15 +445,18 @@ protected BundleContext getContext() {
   /** {@inheritDoc}. */
   @Override
   public List<Map<String, Object>> getPluginsForApplication(String appName) {
-    List<Map<String, Object>> returnValues = new ArrayList<>();
-
-    for (ApplicationPlugin plugin : applicationPlugins) {
-      if (plugin.matchesAssocationName(appName)) {
-        returnValues.add(plugin.toJSON());
+    try {
+      List<Map<String, Object>> returnValues = new ArrayList<>();
+      for (ApplicationPlugin plugin : applicationPlugins) {
+        if (plugin.matchesAssocationName(appName)) {
+          returnValues.add(plugin.toJSON());
+        }
       }
+      return returnValues;
+    } catch (Exception e) {
```

Comment:
Which exceptions is this try/catch trying to catch?

Reply:
This is probably one place where we can catch `RuntimeException` instead. I'll have to check.

Reply:
Wonder if we should just catch throwable instead. I can't think of a scenario when we want any stacktrace to be returned in the jolokia response.

Reply:
If you do so then make sure to first catch `VirtualMachineError` and throw it back without dong anything else.